source/cn/guide/use-vue.md

```diff
@@ -5,7 +5,280 @@ group: Overview
 order: 1.6
 version: 2.1
 ---
+<!-- 
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ -->
 
-Working in progress.
+自2016年2月17日发布WeexSDK [v0.10.0](https://github.com/alibaba/weex/releases/tag/v0.10.0)后,Weex集成了v2版本的Vue。Vue（读音 /vjuː/，类似于 view 的读音）是一套构建用户界面(user interface)的渐进式框架。详情请参阅其[官方网站](https://cn.vuejs.org/)。
```

Comment:
Please use Chinese punctuations. Use `，`  and `。` instead of `,` and `.`. 

By the way, I think it's a good practice to add a space between Chinese words and English words. Such as `Weex 集成了 v2 版本的 Vue。`

As well as the follow-up paragraphs.